### ISSUE TITLE
Can Raccoon be used via command line?

### ISSUE THREAD

**Issue by ahm750** [onyxbits/raccoon4#121]
Hey! Can I download APKs via the command line instead of using the executable? I couldn’t find anything about it.

**Comment by Vall3y** [comment]
@ahm750 yes, I was able to `java -jar raccoon-4.24.0.jar --help` worked for me
